src/commands/enter.rs

```diff
@@ -0,0 +1,40 @@
+use anyhow::{bail, Context, Result};
+use log::info;
+use std::{path::Path, process::Command};
+
+pub fn cmd(instance_name: &str, data_dir: &Path, plugin_path: &Path) -> Result<()> {
+    info!("Entering instance <{instance_name}>");
+
+    let cluster_dir = plugin_path.join(data_dir).join("cluster");
+
+    enter_instance(&cluster_dir, instance_name)
+        .context(format!("failed to enter instance {instance_name}"))?;
+
+    Ok(())
```

Comment:
Можно же прямо так вернуть

```suggestion
    enter_instance(&cluster_dir, instance_name)
        .context(format!("failed to enter instance {instance_name}"))
```